test-setup/scripts/setup-krb5-mongo.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+username=${MONGO_INITDB_ROOT_USERNAME}
+password=${MONGO_INITDB_ROOT_PASSWORD}
+
+echo "Waiting for startup.."
+until mongosh --host 127.0.0.1:27017 -u ${username} -p ${password} --eval 'quit(db.runCommand({ ping: 1 }).ok ? 0 : 2)' &>/dev/null; do
```

Comment:
This will loop forever if it's unable to connect. Shall we consider a timeout or a number of retries?